x-pack/plugins/cloud_security_posture/public/pages/rules/rules_flyout.tsx

```diff
@@ -4,23 +4,165 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import React from 'react';
-import { EuiSpacer, EuiFlyout, EuiFlyoutHeader, EuiFlyoutBody } from '@elastic/eui';
+import React, { useState } from 'react';
+import {
+  EuiSpacer,
+  EuiFlyout,
+  type EuiDescriptionListProps,
+  EuiToolTip,
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiTab,
+  EuiTabs,
+  EuiTextColor,
+  EuiTitle,
+  EuiDescriptionList,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiSwitch,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 import type { RuleSavedObject } from './use_csp_rules';
+import * as TEXT from './translations';
 
-interface FindingFlyoutProps {
+interface RuleFlyoutProps {
   onClose(): void;
+  toggleRule(rule: RuleSavedObject): void;
   rule: RuleSavedObject;
 }
 
-export const RuleFlyout = ({ onClose, rule }: FindingFlyoutProps) => {
+const tabs = ['overview', 'remediation', 'rego code'] as const;
```

Comment:
We might want these internationalized as well, so we need a constant key and an internationalized display value